apps/gp2-server/test/fixtures/event.fixtures.ts

```diff
@@ -6,9 +7,257 @@ import {
+        {
+          nodeType: 'paragraph',
+          data: {},
+          content: [
+            {
+              nodeType: 'text',
+              value: '',
+              marks: [],
+              data: {},
+            },
+            {
+              nodeType: 'embedded-entry-inline',
+              data: {
+                target: {
+                  sys: {
+                    id: 'video-id',
+                    type: 'Link',
+                    linkType: 'Entry',
+                  },
+                },
+              },
+              content: [],
+            },
+            {
+              nodeType: 'text',
+              value: '',
+              marks: [],
+              data: {},
+            },
+          ],
+        },
+      ],
+    },
+    links: {
+      entries: {
+        inline: [
+          {
+            sys: {
+              id: 'video-id',
+            },
+            __typename: 'Media',
+            url: 'https://player.vimeo.com/video/493052720',
+          },
+        ],
+      },
+      assets: {
+        block: [],
+      },
+    },
+  },
+  videoRecordingUpdatedAt: '2010-08-01T08:00:04.000Z',
+  presentationPermanentlyUnavailable: false,
+  presentation: {
+    json: {
+      nodeType: 'document',
+      data: {},
+      content: [
+        {
+          nodeType: 'paragraph',
+          data: {},
+          content: [
+            {
+              nodeType: 'text',
+              value: '',
+              marks: [],
+              data: {},
+            },
+            {
+              nodeType: 'embedded-entry-inline',
+              data: {
+                target: {
+                  sys: {
+                    id: 'presentation-id',
+                    type: 'Link',
+                    linkType: 'Entry',
+                  },
+                },
+              },
+              content: [],
+            },
+            {
+              nodeType: 'text',
+              value: '',
+              marks: [],
+              data: {},
+            },
+          ],
+        },
+      ],
+    },
+    links: {
+      entries: {
+        inline: [
+          {
+            sys: {
+              id: 'presentation-id',
+            },
+            __typename: 'Media',
+            url: 'https://drive.google.com/file/d/1euARcFkYJoqj4c_oor_nYIMBDWOfL-3x/preview',
+          },
+        ],
+      },
+      assets: {
+        block: [],
+      },
+    },
+  },
+  presentationUpdatedAt: '2010-09-01T08:00:04.000Z',
+  meetingMaterialsPermanentlyUnavailable: false,
+  meetingMaterials: [
+    {
+      title: 'My additional link',
+      url: 'https://link.pt/additional-material',
+    },
+  ],
+  calendar: {
+    googleCalendarId: 'c_t92qa82jd702q1fkreoi0hf4hk@group.calendar.google.com',
+    color: '#125A12' as const,
+    name: 'Tech 1 - Sequencing/omics',
+  },
+  thumbnail: {
+    url: 'https://images.ctfassets.net/5v6w5j61tndm/849855829/ac04300efa901ab22278b50d63bfde14/COSA-9900000000079e3c.png',
```

Comment:
not sure these links contain private information, should it be better to use something like http://example.com?